components/SVG/welcome.js

```diff
@@ -1,4 +1,6 @@
-export function WelcomeIcon({ color, width, height }) {
+import { styled } from "styled-components";
```

Comment:
```suggestion
import styled from "styled-components";
```

styled is never imported via destructuring, putting the {__} will cause an error in the console :)